velox/docs/functions/spark/math.rst

```diff
@@ -233,3 +233,11 @@ Mathematical Functions
 .. spark:function:: unaryminus(x) -> [same as x]
 
     Returns the negative of `x`.  Corresponds to Spark's operator ``-``.
+
+.. spark:function:: unhex(x) -> varbinary
+
+    Converts hexadecimal varchar ``x`` to varbinary.
+    If size ``x`` is odd, parses the first character and puts it at index 0 of binary result,
+    then, continues parsing remaining characters, if existing, and
```

Comment:
This behavior is confusing to me. Do I get it right that if 'x' is 1 byte, the result is 'x' unmodified. However, if 'x' is 2n + 1 bytes with n > 0 the first byte is ignored?

Reply:
> Do I get it right that if 'x' is 1 byte, the result is 'x' unmodified. 

decoded `x` first, if decoded result is valid, return decoded result, if invalid, return NULL.

Reply:
> if 'x' is 2n + 1 bytes with n > 0 the first byte is ignored?

decoded first byte, if invalid, return NULL, if valid, keep parsing remaining characters and first byte will be ignore.